src/views/demoDetails/PlayerList/index.tsx

```diff
@@ -83,20 +111,98 @@ export default function PlayerList({ gameSummary }: PlayerListProps) {
+                        ];
+                  scoreboardRef.current?.setScoreboard(scoreboard);
+                }}
+                selected={player.user_id === currentPlayer?.user_id}
+              />
             ))}
           </div>
           <div className={classes.playerListColumn}>
             {redPlayers.map((player) => (
-              <PlayerBox key={player.user_id} player={player} />
+              <PlayerBox
+                key={player.user_id}
+                onClick={() => {
+                  setCurrentPlayer(player);
+                  const scoreboard =
+                    currentTab === "match"
+                      ? player.scoreboard
+                      : player.round_scoreboards[
+                          currentTab as unknown as number
+                        ];
+                  scoreboardRef.current?.setScoreboard(scoreboard);
+                }}
+                player={player}
+                selected={player.user_id === currentPlayer?.user_id}
+              />
             ))}
           </div>
         </div>
       </ScrollArea.Autosize>
+      {/* Divider above the scoreboard*/}
+      <hr className={classes.scoreboardDivider} />
+      <div className={classes.scoreboardPlayerNameHeader}>
+        {currentPlayer?.name ?? ""}
+      </div>
+      <div>
+        <Tabs defaultValue={"match"}>
+          <Tabs.List>
+            <Tabs.Tab
+              value={"match"}
+              onClick={() => {
+                setCurrentTab("match");
+                scoreboardRef.current?.setScoreboard(
+                  currentPlayer?.scoreboard ?? EMPTY_SCOREBOARD
+                );
+              }}
+            >
+              Match
+            </Tabs.Tab>
+            {Array.from({ length: gameSummary.num_rounds }, (x, i) => i).map(
+              (roundNum) => {
+                return (
+                  <Tabs.Tab
+                    value={`round${roundNum}`}
+                    key={roundNum}
+                    disabled={
+                      (currentPlayer?.round_scoreboards[roundNum] ?? null) ===
+                      null
+                    } // disable if the player wasn't in this round
+                    onClick={() => {
+                      setCurrentTab(roundNum);
+                      scoreboardRef.current?.setScoreboard(
+                        currentPlayer?.round_scoreboards[roundNum] ??
+                          EMPTY_SCOREBOARD
+                      );
+                    }}
+                  >
+                    Round {roundNum + 1}
+                  </Tabs.Tab>
+                );
+              }
+            )}
+          </Tabs.List>
+        </Tabs>
+        {/* The actual scoreboard component.  This will be updated when players or tabs are clicked, rather than creating an entirely new component */}
+        <ScoreboardTable
+          ref={scoreboardRef}
+          scoreboard={mainPlayer?.scoreboard ?? EMPTY_SCOREBOARD}
```

Comment:
Instead of showing a bunch of zeros in case `scoreboard` is undefined, we could just skip rendering the scoreboard altogether.
Not sure what is better, let's discuss.

Reply:
This is more to cover the case where something has gone seriously wrong, I don't expect this to actually occur (maybe for demos without a POV player? but I don't have any of those).  In that case, I'd still prefer to display a scoreboard so users will know what to expect in that area of the UI